src/entities/Proposal/routes.ts

```diff
@@ -404,18 +404,39 @@ export async function createProposalTender(req: WithAuth) {
   const user = req.auth!
   const configuration = validate<NewProposalTender>(newProposalTenderValidator, req.body || {})
 
-  if (!PROMOTE_PITCH_ENABLED) {
-    throw new RequestError('Forbidden', RequestError.Forbidden)
-  }
-
   await validateLinkedProposal(configuration.linked_proposal_id, ProposalType.Pitch)
   await validateSubmissionThreshold(user, SUBMISSION_THRESHOLD_TENDER)
 
+  const tenderProposals = await ProposalModel.getProposalList({
+    linkedProposalId: configuration.linked_proposal_id,
+    order: 'ASC',
+  })
+
+  if (
+    tenderProposals.find(
+      (proposal) => proposal.status === ProposalStatus.Passed || proposal.status === ProposalStatus.Rejected
```

Comment:
This is interesting. Is this the only way we have today to know if a pitch proposal finished the tendering process?

Reply:
Yea... We could have something in the pitch configuration? Like a Tender status. Or something in the middle between two pitch/tender proposals.